.github/workflows/IntegrationTestZygote.yml

```diff
@@ -0,0 +1,30 @@
+name: IntegrationTestZygote 
+on:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request:
+
+jobs:
+  test:
+    name: Julia v${{ matrix.julia-version }} - ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1]
+        os: [ubuntu-latest]
```

Comment:
I don't think the matrix is necessary here, since you can just hard code the values, but doesn't really matter,

Reply:
Yeah, it's just there in case we ever want to add other versions/oses, and it currently spawns the same one job we would get if we just hard-coded them.

Reply:
Makes sense!